addons/mass_mailing/static/src/js/mass_mailing_design_constants.js

```diff
@@ -28,7 +28,7 @@ export const PRIORITY_STYLES = {
     'a:not(.btn)': [],
     'a.btn.btn-primary': [],
     'a.btn.btn-secondary': [],
-    'hr': [],
+    'hr': ['border-top-color'],
```

Comment:
This doesn't seem to work. It's being overridden by 
```css
.o_layout .s_hr hr {
    padding: 0;
    border: 0;
    border-top: 1px solid currentColor;
    margin: 0 auto;
    color: inherit;
}
```
Which doesn't use `border-top-color`, but `border-top`, which not only overrides the border's color but also `border-width` and `border-style`.